settings.py

```diff
@@ -52,7 +52,7 @@
 AFTER_HOURS_END = 10
 
 # how old do voters have to be for their vote to count?
-MIN_VOTER_AGE = 1 * 30 * 24 * 60 * 60  # 1 month
+MIN_VOTER_AGE = 12 * 60 * 60  # 1 day (12 hrs)
```

Comment:
@phil-r Yeah, I see how that would be confusing. It's 1 week now, so no worries.